velox/core/Expressions.h

```diff
@@ -320,11 +320,10 @@ class FieldAccessTypedExpr : public ITypedExpr {
 
   std::string toString() const override {
     if (inputs().empty()) {
-      return fmt::format("{}", std::quoted(name(), '"', '"'));
+      return fmt::format("{:?}", name());
```

Comment:
Do we have tests for this code path?

Reply:
I don't know, this is the change recommended by fmt to replace std::quoted.

Reply:
There are no explicit tests.
Can you please include the below diff for completeness?
```
diff --git a/velox/core/tests/TypedExprSerdeTest.cpp b/velox/core/tests/TypedExprSerdeTest.cpp
index 37b402a69..625b599c8 100644
--- a/velox/core/tests/TypedExprSerdeTest.cpp
+++ b/velox/core/tests/TypedExprSerdeTest.cpp
@@ -54,6 +54,7 @@ TEST_F(TypedExprSerDeTest, fieldAccess) {
   std::shared_ptr<ITypedExpr> expression =
       std::make_shared<FieldAccessTypedExpr>(BIGINT(), "a");
   testSerde(expression);
+  ASSERT_EQ(expression->toString(), "\"a\"");
 
   expression = std::make_shared<DereferenceTypedExpr>(
       VARCHAR(),
@@ -61,6 +62,7 @@ TEST_F(TypedExprSerDeTest, fieldAccess) {
           ROW({"a", "b"}, {VARCHAR(), BOOLEAN()}), "ab"),
       0);
   testSerde(expression);
+  ASSERT_EQ(expression->toString(), "\"ab\"[\"a\"]");
 }
```

Reply:
Ah it seems : https://github.com/fmtlib/fmt/issues/3131 this is reason for using :? , but apparently :? is not 1:1 with quoted string ..